ftplugin/org.lua

```diff
@@ -0,0 +1,38 @@
+require('orgmode.config'):setup_mappings('org')
+require('orgmode.config'):setup_mappings('text_objects')
+
+-- options
+vim.opt_local.modeline = false
+vim.opt_local.fillchars:append('fold: ')
+vim.opt_local.foldmethod = 'expr'
+vim.opt_local.foldexpr = 'nvim_treesitter#foldexpr()'
+vim.opt_local.foldtext = 'v:lua.require("orgmode.org.indent").foldtext()'
+vim.opt_local.formatexpr = 'v:lua.require("orgmode.org.format")()'
+vim.opt_local.foldlevel = 0
+vim.opt_local.commentstring = '# %s'
+vim.opt_local.omnifunc = 'function! OrgmodeOmni(findstart, base)'
```

Comment:
Seems like this is not working. We can probably add this to `plugin/orgmode.vim` temporarily until we drop the support for versions < 0.9.0, because 0.9.0 introduced [lua_omnifunc](https://neovim.io/doc/user/lua.html#vim.lua_omnifunc()) which we can leverage here